### ISSUE TITLE
webkit releases socket for iOS10 

### ISSUE THREAD

**Issue by sugatmankar** [google/ios-webkit-debug-proxy#165]
Appium Log :- 

```
[MJSONWP] Calling AppiumDriver.setUrl() with args: ["https://www.google.com...
[debug] [XCUITest] Executing command 'setUrl'
[debug] [iOS] Attempting to set url 'https://www.google.com#'
[debug] [RemoteDebugger] Navigating to new URL: https://www.google.com#
[debug] [RemoteDebugger] Sending WebKit data: **{"method":"Page.navigate"**,"...
[debug] [RemoteDebugger] WebKit remote debugger socket disconnected
```

ios-webkit-debug-proxy log (was started on port 9226):- 

```
Page 9226/0 not found
Already sent close_frame
```

**Comment by artygus** [comment]
did you compile with libimobildevice HEAD?

**Comment by sugatmankar** [comment]
Yes. I have libimobildevice from HEAD. Sometimes it works sometime it
won't. Sometime on Restart machine it works again.

**Comment by artygus** [comment]
@sugatmankar could be fixes with #169, please try to build from head
